Sources/SotoS3FileTransfer/S3FileTransferManager.swift

```diff
@@ -412,8 +413,9 @@ public class S3FileTransferManager {
                 }
                 // construct list of files to delete, if we are doing deletion
                 if delete == true {
+                    let targetKeyMap = Dictionary(uniqueKeysWithValues: targetFiles.map { ($0.to.key, $0) })
```

Comment:
If you are not actually using the values in the dictionary why don't you use a Set instead.

Reply:
Ah of course, good point!